en/docs/chapter_searching/binary_search.md

```diff
@@ -1,24 +1,24 @@
 # Binary search
 
-<u>Binary search</u> is an efficient search algorithm based on the divide-and-conquer strategy. It utilizes the orderliness of data, reducing the search range by half each round until the target element is found or the search interval is empty.
+<u>Binary search</u> is an efficient search algorithm based on the divide-and-conquer strategy. By taking advantage of the orderliness of elements inside the sorted array, it reduces the search interval by half each round until the target element is found or the search interval is empty.
 
 !!! question
 
-    Given an array `nums` of length $n$, with elements arranged in ascending order and non-repeating. Please find and return the index of element `target` in this array. If the array does not contain the element, return $-1$. An example is shown in the figure below.
+    Given an array `nums` of length $n$, where elements arranged in ascending order without duplicates. Please find and return the index of element `target` in this array. If the array does not contain the element, return $-1$. An example is shown in the figure below.
```

Comment:
'where elements arranged in ascending order without duplicates.'
to
'where elements are arranged in ascending order and are without duplicates.'
for better readability and precision.